src/psbt.cpp

```diff
@@ -16,11 +16,6 @@ PartiallySignedTransaction::PartiallySignedTransaction(const CMutableTransaction
     outputs.resize(tx.vout.size());
 }
 
-bool PartiallySignedTransaction::IsNull() const
```

Comment:
I'm pretty sure all of the `PSBT` related `isNull()` methods are used. cc @gwillen.

Reply:
I would believe that they're not. The refactor of PSBT methods went through several iterations and there was definitely dead code at least at one point.

Also, the fact that I can build and run tests with no complaint with the methods removed, suggests they are indeed unused.

More comments below, though.

Reply:
Both `PSBTInput::IsNull()` and `PSBTOutput::IsNull()` were globally unused and untested prior to this PR. In the updated PR they are tested and thus used from the tests at least :-)